api/v1alpha1/instancegroup_types.go

```diff
@@ -224,6 +226,12 @@ type MixedInstancesPolicySpec struct {
 	InstanceTypes []*InstanceTypeSpec `json:"instanceTypes,omitempty"`
 }
 
+type LaunchTemplatePlacementSpec struct {
+	AvailabilityZone     string `json:"availabilityZone,omitempty"`
+	HostResourceGroupArn string `json:"hostResourceGroupArn,omitempty"`
+	Tenancy              string `json:"tenancy,omitempty"`
```

Comment:
Are these the minimal values required to define placement? Are they all required? is it possible to just provide `Tenancy` or other fields?

Reply:
These are the only 3 of 8 fields _supported_ by AutoScalingGroups.
`AvailabilityZone` is optional.
`Tenancy` must be `host` if `HostResourceGroupArn` is set; otherwise they're completely optional.

Reply:
In that case, we should probably add validations accordingly for both `Tenancy` (can only be one of allowed values), and `HostResourceGroupArn` (tenancy must be `host` and value must be ARN).